### PR TITLE
Bug fix and keepalive request

### PR DESCRIPTION
**Pull request by as-j**

### CODE REVIEW

File: src/config.c [hunk 1]
```diff
@@ -68,6 +68,7 @@ static void default_config(void) {
 	conf.mtu = 1500;
 	conf.mode = MODE_TAP;
 	conf.iface_persist = true;
+    conf.keepalive_time = KEEPALIVE_TIMEOUT;
```
Comment (khwaabd) [inline src/config.c]: spaces?

File: src/send.c [hunk 2]
```diff
@@ -184,6 +184,17 @@ void fastd_send_handshake(const fastd_socket_t *sock, const fastd_peer_address_t
 	send_type(sock, local_addr, remote_addr, peer, PACKET_HANDSHAKE, buffer, 0);
 }
 
+/** Sends a keepalive_req packet */
+void fastd_send_keepalive_request(const fastd_socket_t *sock, const fastd_peer_address_t *local_addr, const fastd_peer_address_t *remote_addr, fastd_peer_t *peer) {
+	fastd_buffer_t buffer = fastd_buffer_alloc(0, conf.min_encrypt_head_space, conf.min_encrypt_tail_space);
```
Comment (khwaabd) [inline src/send.c:189]: this is freed by send_type? correct?

**Review by khwaabd — APPROVED** [review]

File: src/config.c (hunk 1 above)
Comment (as-j) [reply src/config.c]: fixed it

File: src/send.c (hunk 2 above)
Comment (as-j) [reply src/send.c:189]: Correct

**Pull request merged**